### PR TITLE
Make changes to privacy public

### PR DESCRIPTION
**Pull request by karussell**
Also improve privacy for more transparency required by GDPR

### CODE REVIEW

File: privacy.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+misusage of API tokens and to
+improve performance and usability of our API.
+Our customers have the choice to use a proxy to enhance the privacy of their
+users.
+</p>
+
+<p>
+Customers can choose between different geocoding providers. If "opencagedata" is selected, requests go to
+<a href="https://geocoder.opencagedata.com/gdpr">OpenCageData</a>. If "gisgraphy" is selected, requests go to
+<a href="https://www.gisgraphy.com/privacy">Gisgraphy</a>.
```
Comment (boldtrn) [inline privacy.html]: What about nominatim?

File: privacy.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+</p>
+
+<p>
+Every API request is stored. We save the associated information (request body and
+headers, IP, time) for a maximum of 5 weeks. We use this to identify
+misusage of API tokens and to
+improve performance and usability of our API.
+Our customers have the choice to use a proxy to enhance the privacy of their
+users.
+</p>
```
Comment (boldtrn) [inline privacy.html]: Should we mention Kurviger here as well? Probably not necessary?

File: privacy.html (hunk 1 above)
Comment (karussell) [reply privacy.html]: This is hosted by Opencagedata in our case

File: privacy.html (hunk 2 above)
Comment (karussell) [reply privacy.html]: Hmmh, as this is an individual, separate contract I don't think we should include it here.

File: privacy.html (hunk 1 above)
Comment (karussell) [reply privacy.html]: Ah, but yes. You are right. We need to add the case nominatim here.

Comment (boldtrn) [reply privacy.html]: > Ah, but yes. You are right. We need to add the case nominatim here.

Yes, that's what I meant :)

**Comment by karussell** [comment]
Put them live and will now switch to new terms version in dirigent.

**Comment by oblonski** [comment]
Cool. Thanks a lot, @karussell. This is also smth for our March Newsletter.

**Pull request merged**